### DISCUSSION TITLE
Welcome to NovelScraper Discussions!

### DISCUSSION THREAD

**Discussion by dr-nyt** [HanaDigital/NovelScraper#53] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by thejainlabs** [comment]
can we please have the ability to only download selected chapters. I have ready 3000 chapters of supreme magus and only want to download last 100 chapters, but have to download all 3000+ chapters after every update in website. It is an ongoing website. 
thanks a lot :)

**Reply by dr-nyt** [reply]
Did you happen to delete the download folder? Normally it is supposed to update the existing file and will only download the new chapters. Or it could be that there is a bug, I'll look into it.

**Reply by thejainlabs** [reply]
No no, I think you got it wrong. I read the supreme magus chapters on a website. Then I came upon this tool. So I just wanted to download the last 100/200 chapters or may be have the have the ability o download the selected range of chapters that I want. Is that possible?

**Reply by thejainlabs** [reply]
By the way. it is a great tool no doubt about that.

**Reply by dr-nyt** [reply]
@thejainlabs I just released version 3 which speeds up downloads by 5 times, so this should help you:
[https://github.com/HanaDigital/NovelScraper/releases/latest](https://github.com/HanaDigital/NovelScraper/releases/latest)
